File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

bool solution(const char* s) {
    bool answer = true;
    int i=0;
    while(s[i])
    {
        if(s[i] < 48 || s[i] > 57)
        {
            answer = false;
            break;
        }
        i++;
    }
    if((i == 4 || i == 6) && answer){answer = true;}
    else{answer = false;}
    return answer;
}

File: 프로그래머스/1/12918.문자열 다루기 기본-Python3/문자열 다루기 기본.py
def solution(s):
    answer = True
    if len(s) in [4, 6]:
        for c in s:
            if c in ['0','1','2','3','4','5','6','7','8','9']:
                pass
            else:
                answer = False
                break
    else:
        answer = False
    
    return answer